libsolidity/experimental/codegen/IRGenerator.cpp

```diff
@@ -72,8 +72,12 @@ std::string IRGenerator::run(
 	Whiskers t(R"(
 		object "<CreationObject>" {
 			code {
-				codecopy(0, dataoffset("<DeployedObject>"), datasize("<DeployedObject>"))
-				return(0, datasize("<DeployedObject>"))
+				<?eof>
+					returncontract("<DeployedObject>", 0, 0)
```

Comment:
Shouldn't auxdata here involve the immutables?

Reply:
Nevermind, I just noticed that this is in ``libsolidity/experimental`` (that's currently largely unmaintained)